linkerd/app/outbound/src/http/detect.rs

```diff
@@ -31,6 +43,8 @@ impl<N> Outbound<N> {
         self.map_stack(|config, rt, tcp| {
             let ServerConfig { h2_settings, .. } = config.proxy.server;
 
+            let tcp = tcp.instrument(|_: &_| debug_span!("opaque"));
```

Comment:
nit: looks like the `opaque` span is being added to the TCP stack that's used if detection is disabled, _or_ if HTTP is not detected...i feel like it might be clearer if there were separate "opaque" and "tcp" spans to differentiate between "no HTTP detected" and "HTTP detection skipped" cases?

Reply:
i'd rather not vary that at the span level. we have events on the parent that indicate any detection timeouts.